gno.land/pkg/gnoweb/components/ui/logo.html

```diff
@@ -4,42 +4,52 @@
     <path
       d="M15.7782 18.1953C15.5729 17.4557 15.1344 16.8048 14.5394 16.3075C14.3097 16.1152 14.0591 15.9332 13.7854 15.7636C13.5534 15.6191 13.261 15.8319 13.3283 16.0936L13.493 16.7331C13.7018 17.5456 12.8133 18.2044 12.0732 17.7868L10.0838 16.6648C8.77648 15.9275 7.16872 15.9275 5.8614 16.6648L3.872 17.7868C3.13192 18.2044 2.24336 17.5444 2.45216 16.7331L2.62152 16.0754C2.6888 15.8148 2.39764 15.602 2.16564 15.7454C1.86172 15.932 1.58564 16.1346 1.33508 16.3474C0.749278 16.8458 0.342118 17.5149 0.151878 18.2522L0.144918 18.2795C-0.338802 20.1639 0.412879 22.145 2.03456 23.2612L7.05736 26.717C7.6072 27.0948 8.33916 27.0948 8.889 26.717L13.9118 23.2612C15.5578 22.129 16.3072 20.1047 15.7782 18.1965V18.1953Z"
       fill="#999999"
+      class="dark:[fill:#949494]"
```

Comment:
Why not using `fill` utility and merging the `fill` attribute above in one regular `class` attr instead?